datavalidation/src/main/java/org/jumbune/datavalidation/DataValidationMapper.java

```diff
@@ -51,6 +56,10 @@
 	
 	/** The slave file loc. */
 	String SLAVE_FILE_LOC = "slaveFileLoc";
+	
+	/* Logger */
+	private static final Logger LOGGER = LogManager
```

Comment:
Inducing a custom logger here requires logger jar to be add to container JVM's. Are we doing that? Preferably, we should avoid it.